### DISCUSSION TITLE
Exclude Term (inverse-match)

### DISCUSSION THREAD

**Discussion by madmaxmatze** [krisk/Fuse#632] [Ideas]
**Motivation**
I would like to exclude all list items containing a certain term

**Proposal**
Add a "-" operator to https://fusejs.io/examples.html#extended-search
Example: "-php" would return all items which do not contain the term "php" in any of the fields at any location

Thanks Kiro!!

**Comment by krisk** [comment]
Isn't this already handled by inverse-exact-match operator? so `!php`?
